client/template.js

```diff
@@ -1,4 +1,6 @@
 module.exports = async(name, title = '', props = {})=>{
+	const HOMEBREWERY_PUBLIC_URL=(props.publicUrl ? props.publicUrl : 'https://homebrewery.naturalcrit.com');
+
```

Comment:
What if we move this check out to `app.js` line 267? Then we can just pass in the correct value from the beginning?

Reply:
Also, can we set up some default values in the config files?